src/Task/PhpCsFixerV2.php

```diff
@@ -82,15 +89,23 @@ public function run(ContextInterface $context)
         }
 
         $arguments->addOptionalArgument('--using-cache=%s', $config['using_cache'] ? 'yes' : 'no');
-        $arguments->addOptionalArgument('--path-mode=%s', $config['path_mode']);
+        $arguments->addOptionalArgument('--path-mode=intersect', $config['can_intersect']);
```

Comment:
We currently don't have a system to do e2e tests with 3rd party tools. I looked over that one during commit and will test these changes thoroughly before merging.